CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # PayPal iOS SDK Release Notes
 
+## Unreleased
```

Comment:
nit: [CI looks for lowercased unreleased specifically](https://github.com/paypal/paypal-ios/blob/84542b0fcdc5bc48bceacb1d0184c99d1120163f/.github/workflows/release.yml#L20)
```suggestion
## unreleased
```